offline_imu_cam/data_loading.hpp

```diff
@@ -24,7 +24,6 @@ class lazy_load_image {
 	{ }
 	std::unique_ptr<cv::Mat> load() const {
 		auto img = std::unique_ptr<cv::Mat>{new cv::Mat{cv::imread(_m_path, cv::IMREAD_GRAYSCALE)}};
-		/* TODO: make this load in grayscale */
 		assert(!img->empty());
 
 		// Sam's note: I am moving cvtColor into slam2
```

Comment:
One more request :laughing: :
Delete this comment and the commented code; it is now out-of-date.

Reply:
You got it